Local.md

```diff
@@ -1,94 +1,121 @@
+
+## Programming tools  🛠
+|Name|Details|
+|--|--|
+|[Github Desktop](https://desktop.github.com)|Easy GUI for using git without terminal commands| 
+|[Postico](https://eggerapps.at/postico/)|GUI for Postgres databases|
+|[Chrome](https://www.google.com/intl/en/chrome/)|Used in development|
+
+## Code Editor
+Any of these code editors are recommended.
+* [Atom](https://atom.io)
+* [SublimeText ](https://www.sublimetext.com) :heavy_dollar_sign:
+* [VSCode](https://code.visualstudio.com)
+
+#### :electric_plug:Plugins
+
+* `Tabnine` - An AI tool that predicts what you're going to type!
+* `Rubocop autocorrect` - A code analyser and format checker to make sure your code look good
+
+## Communication :telephone_receiver:
+|Name|Details|
+|--|--|
+|[Slack](https://slack.com/intl/en-ie/)|The main way to communicate with the team| 
+|[Apple calendar](https://apps.apple.com/us/app/calendar/id1108185179)| To have all your meetings and engagements in one place|
+|[Fantastical](https://apps.apple.com/us/app/fantastical-calendar-tasks/id718043190) :heavy_dollar_sign:|A calendar alternative|
+|[Apple Mail](https://apps.apple.com/us/app/mail/id1108187098)|For emails|
+|[Superhuman](https://superhuman.com) :heavy_dollar_sign:|Fast and seamless mail|
+|[Loom](https://www.loom.com/)|To create and share screen recordings|
+|[Around](https://www.around.co)|A fun video calling tool|
+|[Zoom](https://zoom.us)|For video call, when Around doesn't work|
+
+## Organisation  :nerd_face:
+|Name|Details|
+|--|--|
+|[Apple notes](https://support.apple.com/en-ie/HT205773)|To jot down quick notes| 
+|[Bear](https://bear.app) :balance_scale:|Note taking app with additional functionality|
+|[Apple Reminders](https://support.apple.com/en-ie/HT205890)|To keep track of tasks|
+|[1 Password](https://1password.com) :heavy_dollar_sign:|Creates and stores strong passwords|
+|[Swish](https://highlyopinionated.co/swish/) :heavy_dollar_sign:|To control windows using trackpad gestures|
+
+
+## Online tools  🛠
+|Name|Details|
+|--|--|
+|[Google Docs](https://www.google.com/docs/about/)|Easy collaboration writing documents| 
+|[Google Slides](https://www.google.com/sheets/about/)|Easy collaboration making powerpoints|
+|[Google Sheets](https://www.google.com/slides/about/)|Easy collaboration making spreadsheets|
+|[Dropbox Paper](https://www.dropbox.com/paper)|Similar to Google Docs|
+|[Figma](https://www.figma.com)|Wireframing tool for designing apps|
+|[Miro](https://www.miro.com)|An online collaborative whiteboard platform|
+|[Jira](https://www.atlassian.com/software/jira)|Project management tool for teams|
+|[Linear](https://linear.app)|Helps streamline software projects, sprints and tasks|
+|[Github](https://www.github.com)|To track code changes and manage versions|
+
+
+## Mobile Apps  :iphone:
+It will be useful to get the mobile version for some of the above apps also.
+
+* Calendar
+* Email
+* Authenticator app 2FA
 * Slack
 * Linear
-* Around 
+* Around
 * Zoom
```

Comment:
link these apps and add reasons for why this would be important to have on mobile. and give option to how they can reduce notifications here. so for slack, you can have different mobile notification settings where mobile only notified when you are offline on the desktop and when someone mentions you - meaning its possibly important to look at.